lib/modules/showImages.js

```diff
@@ -72,14 +72,14 @@ module.options = {
 	},
 	maxWidth: {
 		type: 'text',
-		value: '640',
-		description: 'Max width of image displayed onscreen (enter zero for unlimited).',
+		value: '100%',
+		description: 'Max width of media (in pixels, enter zero for unlimited). Percentage of window width may also be used (e.g. "100%").',
 		advanced: true,
 	},
 	maxHeight: {
 		type: 'text',
-		value: '0',
-		description: 'Max height of image displayed onscreen (enter zero for unlimited).',
+		value: '95%',
```

Comment:
Hmm, 95% is just tall enough that images with titles or descriptions require scrolling, and I think we should be a little conservative since we're increasing it from 640x480.

Maybe bring this down to 75 or 80%?